app/views/interview_surveys/new.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>InterviewSurveys#new</h1>
+<p>Find me in app/views/interview_surveys/new.html.erb</p>
+<br>
+<% if Feature.enabled?(:survey_feature, current_user) %>
+  <p>My feature is enabled!</p>
+<% end %>
```

Comment:
I'm thinking we should wrap the entire form in the feature enabled check. We wouldn't want a user to see anything in the case of ending up on this page.

Also - this feels like an opportunity for a test. Checking that the form is present when the feature is enabled and checking that the form is not present (e.g. checking for the redirect) when the feature is not enabled.

Reply:
I'm adding a single input form to show it being wrapped by the flag.